src/controllers/expenses.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable space-before-function-paren */
+'use strict';
+
+const { getUserById } = require('../services/users');
+const {
+  getAll,
+  getOne,
+  create,
+  remove,
+  update,
+  normalize,
+} = require('../services/expenses');
+
+const getAllExpenses = async (req, res) => {
+  const { userId, from, to, categories } = req.query;
+  let preparedExpenses = await getAll();
+
+  if (userId) {
+    preparedExpenses = preparedExpenses
+      .filter(expense => expense.userId === +userId);
+  }
+
+  if (from) {
+    const fromDate = new Date(from);
+
+    preparedExpenses = preparedExpenses.filter(expense =>
+      new Date(expense.spentAt) > fromDate
+    );
+  }
+
+  if (to) {
+    const toDate = new Date(to);
+
+    preparedExpenses = preparedExpenses.filter(expense =>
+      new Date(expense.spentAt) < toDate
+    );
+  }
+
+  if (categories) {
+    preparedExpenses = preparedExpenses
+      .filter(expense => expense.category === categories);
+  }
+
+  res.send(preparedExpenses.map(expense => normalize(expense)));
+};
+
+const getExpenseById = async (req, res) => {
+  const { expenseId } = req.params;
+
+  const foundExpense = await getOne(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(normalize(foundExpense));
```

Comment:
It would be great if you send status here too.
You could chaing this methods `res.sendStatus(statusCode).send(info)`
for example here your statusCode can be 200 and info can be normalized object